### PR TITLE
Zeek blog: dns anomalies, part 1, fix duplicate meta description

### PR DESCRIPTION
**Pull request by mkorkalo**

### CODE REVIEW

File: _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
+---
```
Comment (esatormi) [inline _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md:1]: Run file through prettier, command is in README.md. Then its nicely readable also in editor.

File: _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+* Edit that file with your favorite editor and disable some modules:
+  ```
+  vim /opt/zeek/share/zeek/base/init-default.zeek
+  ```
+
+* If you wanted to disable all of them, you’d comment out the _@load_ lines with the following vim command:
+  ```
+  _:%s/^@load/#@load/g_
+  ```
+* Then save and quit using
```
Comment (esatormi) [inline _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md:99]: missing ":" from end

File: _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+  ```
+  :wq 
+  ```
+* Restart Zeek
```
Comment (esatormi) [inline _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md:103]: same here

File: _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+  ```
+  /etc/init.d/zeek restart
+  ```
+* Keep disabling zeek modules until you’ve reached the desired noise level in your event log.
+
+### SensorFleet environment
+
+* Navigate to Zeek Configuration menu (Zeek Instrument -> Configure -> Custom configuration)
```
Comment (esatormi) [inline _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md:111]: "." to the end

File: _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+## Step 4: Add script to detect DNS anomalies
+
+### Generic environment
+
+* Install the dns-anomalous package using Zeek Package Manager:
+  ```
+  zkg install sensorfleet/anomalous-dns
+  ```
+
+* Restart Zeek instance (command may depend on installation)
```
Comment (esatormi) [inline _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md:133]: :

File: _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+* Copy the contents of the [anomalous-dns script](http://sensorfleet.com/misc/anomalous_dns_bundle.zeek) and paste it into the text field. Type some name for the script, and validate the syntax. Finally, click Save.
+
+* Click Commit script changes from the IDS Rule Manager UI
+  <img src="/img/posts/zeek_1_blog/09_commit.png" title="Commit script changes" width=500>
+
+* The Zeek instance will restart with the new script shortly.
+
+## Step 5: Configure DNS anomalies whitelist
+
+* Navigate to Zeek events list to see if it’s producing any events. Depending on the network and port mirroring configuration, you might see false positives in the event output. For SensorFleet environment, use the UI. For generic environment, watch zeek’s _notice.log_.
```
Comment (esatormi) [inline _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md:164]: capitalize zeek

File: _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+
+Also, some malwares might use DNS by default, even if HTTP was allowed, because DNS might be harder to detect since they do not make direct connections to the outside world, but instead leverage your DNS resolver.
+
+I found out that configuring the whitelist for nearly zero false positives can take a lot of work, though, depending on your network. That’s why I ended up patching the [jbaggs/anomalous-dns](https://github.com/jbaggs/anomalous-dns) for a more aggressive whitelist.
+
+We now use the above detection method in monitoring of our own infrastructure.
+
+Hopefully this will help someone to realize how neat things you can do just by using open source tools such as Zeek.
+
+It gets even easier when using commercial products to round the rough edges a bit.
```
Comment (esatormi) [inline _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md:219]: "  ... source tools such as Zeek. And it gets even easier..."

Combining one-sentence paragraphs.

File: _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+  ```
+* Then save and quit using
+  ```
+  :wq 
+  ```
+* Restart Zeek
+  ```
+  /etc/init.d/zeek restart
+  ```
+* Keep disabling zeek modules until you’ve reached the desired noise level in your event log.
```
Comment (esatormi) [inline _posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md:107]: capitalize zeek

File: _includes/head.html [hunk 9]
```diff
@@ -48,8 +48,14 @@
   {% else %}
   <meta property="og:title" content="SensorFleet" />
   <title>SensorFleet</title>
-  {% endif %} {% if page.description %}
+  {% endif %}
+  {% if page.description %}
   <meta name="description" content="{{ page.description }}" />
+  {% else %}
+  <meta
```
Comment (esatormi) [inline _includes/head.html:55]: Not a good idea according to internets:

"If all of your pages have similar meta descriptions, they will be more misleading than helpful.

If you don’t have time to write meta descriptions for every page, you can let Google generate them automatically. Based on the query, Google will pull relevant parts of your page’s content to feature in the snippet."

Comment (mkorkalo) [reply _includes/head.html:55]: That was there before, this modification just removes the duplicate.
You think we should remove the default description totally? What pages are affected, should we do that in another mr and leave the blog be?

**Pull request merged**